### PR TITLE
[s]

### PR DESCRIPTION
**Pull request by alexkar598**
E

### CODE REVIEW

**Comment by monster860** [comment]
what does this even fix

**Comment by yacabo** [comment]
10/10 documentation

**Review by monster860 — CHANGES_REQUESTED** [review]
what does this fix

**Comment by AutisticFroggy** [comment]
I raised this issue in msay after retaining knowledge of the bug for some time, I didn't think it was too harmful until we discovered a method to abuse it for infinite metal glitches, then reported it to alexkar. This bug is in reinforced floors that causes efforts to remove the tile with anything except a wrench fail, this means singularity gravity_act() procs and disposals underfloor item release. It still releases metal rods during the process, but does not delete, causing an infinite amount of metal rods to issue forth from a single rfloor. I'm pretty sure this should fix it.

File: code/game/turfs/simulated/floor/reinf_floor.dm [hunk 1]
```diff
@@ -33,6 +32,9 @@
 /turf/open/floor/engine/try_replace_tile(obj/item/stack/tile/T, mob/user, params)
 	return
 
+/turf/open/floor/engine/remove_tile(mob/user, silent = FALSE, make_tile = TRUE)
+	return
```
Comment (Amelia0010) [inline code/game/turfs/simulated/floor/reinf_floor.dm:36]:
```suggestion
	.=..()
	ScrapeAway(flags = CHANGETURF_INHERIT_AIR)
```

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: code/game/turfs/simulated/floor/reinf_floor.dm (hunk 1 above)
Comment (alexkar598) [reply code/game/turfs/simulated/floor/reinf_floor.dm:36]: No that makes it be stupid and counter the point of reinforced floors being nearly indestructible by any means

**Comment by alexkar598** [comment]
Let's not talk about unpatched exploits in public, ya?

**Pull request merged**